README.md

```diff
@@ -1,12 +1,14 @@
-# CAPTCHA plugin
+# Kirby3 captcha
 
-A plugin for [Kirby CMS](http://getkirby.com) to secure forms with CAPTCHAs using [Securimage](http://www.phpcaptcha.org)
+**Requirement:** Kirby 3.0
+
+A plugin for [Kirby3 CMS](http://getkirby.com) to secure forms with CAPTCHAs using [Securimage](http://www.phpcaptcha.org)
 
 ## Installation
 
 Clone this repository into `/site/plugins`:
 
-`git clone --recursive https://github.com/bjo3rnf/kirby-plugin-captcha captcha`
+`git clone --recursive https://github.com/marsch-/kirby3-captcha captcha`
```

Comment:
I suggest to either keep the original GitHub URL or to further develop the plugin as a fork and use yours.

Reply:
I'm happy if you continue it within your repository, people will most likely expect it there. I however suggest to fork a Kirby3 version and rename it after the convention most of the plugins adhere now: https://github.com/search?q=kirby3